language/move-prover/interpreter/src/concrete/player.rs

```diff
@@ -489,7 +489,7 @@ impl<'env> FunctionContext<'env> {
                     typed_args.remove(0),
                     global_state,
                 )
-                .map(|_| vec![])
+                .map(|_| Vec::new())
```

Comment:
Is this a lint? I don't like this one, but not a strong opinion.

Reply:
Yeah linter complained to me here

Reply:
Yeah this one also feels odd to me... Since we're using the `vec![]` syntax for non-empty lists universally, requiring the empty case to be written in a different way seems like a very fractured experience to me.